.github/workflows/trigger-gitlab.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
       SCHUTZBOT_SSH_KEY: ${{ secrets.SCHUTZBOT_SSH_KEY }}
-      SKIP_CI: ${{ github.event.pull_request.draft == true || contains(github.event.pull_request.labels.*.name, 'WIP') }}
+      SKIP_CI: ${{ github.event.pull_request.draft == true || contains(github.event.pull_request.labels.*.name, 'WIP') && !contains(github.event.pull_request.labels.*.name, 'WIP+test') }}
```

Comment:
Can you add parentheses to the condition? It's hard to parse whether `cond1 || cond2 && cond3` is `(cond1 || cond2) && cond3`, or `cond1 || (cond2 && cond3)`.

Reply:
` SKIP_CI: ${{ (github.event.pull_request.draft == true || contains(github.event.pull_request.labels.*.name, 'WIP')) && !contains(github.event.pull_request.labels.*.name, 'WIP+test') }}` Is that correct? I'm not sure in case of github actions.

Reply:
Yeah, that sounds correct. I even wonder if we should support the `WIP` label, marking a PR as a draft should be enough. No strong opinion though.